ca/server.go

```diff
@@ -90,6 +93,13 @@ func NewServer(store *store.MemoryStore, securityConfig *SecurityConfig, rootCAP
 	}
 }
 
+// ExternalCA returns the current external CA
+func (s *Server) ExternalCA() *ExternalCA {
```

Comment:
Please add a comment saying this is exposed to support unit testing and `externalCA` is nominally a private field.